src/main/java/ro/societateahermes/backendservice/controller/TimelineControllerInterface.java

```diff
@@ -0,0 +1,20 @@
+package ro.societateahermes.backendservice.controller;
+
+import ro.societateahermes.backendservice.entities.Event;
+import ro.societateahermes.backendservice.entities.dto.ActivityDTO;
+import ro.societateahermes.backendservice.entities.Timeline;
+
+import java.util.List;
+
+public interface TimelineControllerInterface {
+
+    Timeline createActivityFromTimeline(Long IdEvent, ActivityDTO activityDTO);
+
+    Timeline updateActivityFromTimeline(Long IdEvent, ActivityDTO activityDTO);
+
+    void deleteActivityFromTimeline(Long IdEvent, ActivityDTO IdActivity);
+
+    ActivityDTO getOneActivityFromTimeline(Long IdEvent, ActivityDTO activityDTO);
+
+    List<ActivityDTO> getAllActivityFromTimeline(Timeline timeline);
```

Comment:
Ar trebui sa avem si un mod prin care sa obtinem un timeline in functie de id-ul eventului pentru care e site-ul. Faci pls un endpoint si pentru asta?